model_analyzer/cli/model.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This file is named `model.py`. We have a `model.py` in `model_analyzer/monitor/` as well as `model_analyzer/triton/model/`. Is there a reason for this convention? I think its a little confusing. This file can be named `cli.py` and the other one `monitor.py`

Reply:
I was trying to follow David Zier's comments for the base classes. We can use `_base.py` if this is less confusing. Or I can use the same name for as the package folder.

Reply:
I assumed that was a particular example, rather than a general rule to call the base class `model`. I also believe it may have been a typo (I could be wrong about that). I think since we have the one class per file rule, we should name the file corresponding to the class inside.

Reply:
Got it, changed the file name to `cli.py`.